deepspeed/runtime/engine.py

```diff
@@ -2686,6 +2693,12 @@ def _load_checkpoint(self,
         if checkpoint is None:
             return None, None
 
+        fetch_z3_params = False
+        if self.zero_optimization_partition_weights() and not load_optimizer_states:
+            from deepspeed.utils.zero_to_fp32 import get_fp32_state_dict_from_zero_checkpoint
```

Comment:
Any reason to have this import here instead of at the top?

Reply:
No good reason. Will move to the top.